azkaban-common/src/main/java/azkaban/project/JdbcProjectImpl.java

```diff
@@ -1070,6 +1070,20 @@ public boolean isFlowFileUploaded(final int projectId, final int projectVersion)
     return !data.isEmpty();
   }
 
+  @Override
+  public boolean isFlowInProject(String projectName, String flowId) {
+    Project project = fetchProjectByName(projectName);
+    try {
+      List<Flow> flow = dbOperator.query(ProjectFlowsResultHandler.SELECT_PROJECT_FLOW,
+          new ProjectFlowsResultHandler(), project.getId(),
```

Comment:
Project can be null if it does not exist. It will result in NPE here. Please handle that case.

Reply:
thanks for the catch. added